src/palindrom.js

```diff
@@ -632,17 +641,35 @@ var Palindrom = (function () {
   };
 
   Palindrom.prototype.observe = function () {
-    this.observer = this.jsonpatch.observe(this.obj, this.filterChangedCallback.bind(this));
+    /* if we haven't ever proxified our object,
+    this means it's the first observe call,
+    let's proxify it then! */
+    if (!this.isObjectProxified) {
+        this.obj = this.jsonPatcherProxy.observe(true, this.filterChangedCallback.bind(this));
+        this.isObjectProxified = true;
+    }
+    /* we are already observing, just enable event emitting. */
+    else {
+        this.jsonPatcherProxy.switchObserverOn();
```

Comment:
It would be cool to enable observer on any object separately. I mean:

```js
this.obj.switchObserverOn();
```

or

```js
this.jsonPatcherProxy.switchObserverOn(this.obj);
```